internal/namespaces/baremetal/v1/testdata/test-baremetal-get-offer-simple.golden

```diff
@@ -0,0 +1,183 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟩🟩🟩 STDOUT️ 🟩🟩🟩️
+ID                  c5853302-63e4-40c7-a711-4a91629565c8
+Name                EM-L105X-SATA
+Stock               available
+Bandwidth           1000000000
+CommercialRange     lithium
+PricePerHour        € 0.213
+Enable              true
+QuotaName           EM_L105X_SATA
+SubscriptionPeriod  hourly
+OperationPath       -
+Fee                 € 0.00
+PrivateBandwidth    1000000000
```

Comment:
Can we improve bandwidth to have a readable format ?

Reply:
Change to Mbit after discussion